File: PersonSpaceshipsGame/Models/Cards/Person/PersonCard.cs
using PersonSpaceshipsGame.Models.Players;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Diagnostics.CodeAnalysis;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static PersonSpaceshipsGame.Models.Cards.Enums;

namespace PersonSpaceshipsGame.Models.Cards.Person
{
    public class PersonCard : IPersonCard
    {
        #region Properties
        public Guid Id { get; set; }
        public float Mass { get; set; }
        public string Name { get; set; }

        [Column("CardType")]
        public string CardTypeString
        {
            get { return CardType.ToString(); }
            private set { CardType = Enums.ParseEnum<CardType>(value); }
        }

        [NotMapped]
        public Enums.CardType CardType { get; set; } = Enums.CardType.Person;

        [NotMapped]
        public Player Player { get; set; }
        #endregion

    }
}

File: PersonSpaceshipsGame/ClientApp/src/app/models/cards/PersonCard.ts
import { PlayableCard } from './PlayableCard';
import { Player } from '../players/Player';
import { CardType } from './CardType';

export class PersonCard extends PlayableCard {
  cardType: CardType;
  id: string;
  name: string;
  player: Player;
  mass: number;

  constructor(id: string, name: string, player: Player, mass: number) {
    super(id, name, player);
    this.mass = mass;
  }

  getPower() {
    return this.mass;
  }
}
